shared/templates/create_sysctl.py

```diff
@@ -33,8 +33,19 @@ def get_files_for_id(self, var_id):
         }
 
     def generate(self, target, serviceinfo):
+
         # get the items out of the list
-        sysctl_var, sysctl_val = serviceinfo
+        # items can be in format
+        # <sysctl_parameter_name, value> or
+        # <sysctl_parameter_name, value, type>
+        try:
+            sysctl_var, sysctl_val = serviceinfo
+            # Default data type for sysctl is int
+            data_type = "int"
+        except ValueError as e:
+            if str(e) == "too many values to unpack (expected 2)":
```

Comment:
Maybe I'm being overly precautious here.

Reply:
I would go for `if len(serviceinfo) == 3`, the examination of the exception error message may change, and four values would have the same message as three.

Reply:
Fair enough, updated per suggestion.

Reply:
Also dealt with exception case, when CSV line is out of the norm